static/app/components/feedback/feedbackItem/feedbackItem.tsx

```diff
@@ -79,6 +80,10 @@ export default function FeedbackItem({feedbackItem, eventData, tags}: Props) {
         <Section icon={<IconTag size="xs" />} title={t('Tags')}>
           <TagsSection tags={tags} />
         </Section>
+
+        <Section icon={<IconChat size="xs" />} title={t('Comments')}>
```

Comment:
Lets rename to Activity to align with the issues details page:
<img width="395" alt="SCR-20240108-mzbn" src="https://github.com/getsentry/sentry/assets/187460/441b6d09-627c-434f-886c-a5d21441c7c0">